frontend/src/scenes/paths/Paths.tsx

```diff
@@ -43,7 +43,9 @@ export function PathsComponent({ nodeCard }: PathsComponentProps): JSX.Element {
     useEffect(() => {
         setNodeCards([])
 
-        const elements = document?.getElementById(id)?.querySelectorAll(`.Paths svg`)
+        // Remove the existing SVG canvas(es). The .Paths__canvas selector is crucial, as we have to be sure
+        // we're only removing the Paths viz and not, for example, button icons.
```

Comment:
Already discovered one weird bug thanks to these tests: turns out resizing the window or toggling the sidebar resulted in the User Paths node card kebab button (dropdown) being removed 🤔

Reply:
Haha, good catch!